src/agent/testresultreader.ts

```diff
@@ -145,6 +145,7 @@ export class ResultReader {
 
                 //testcase duration
                 var testCaseDuration = 0; //in seconds
+                var testCaseDurationInMs = 0;
```

Comment:
Oversight on my part, thought of doing it but ended up not using the variable. Will delete it.